### DISCUSSION TITLE
Low speed

### DISCUSSION THREAD

**Discussion by array05** [norkunas/youtube-dl-php#180] [Q&A] [answered]
Thanks a lot for the previous answers, there was another question with the speed of downloading videos by link, sometimes the download speed is fast, but most often it's somewhere around 200kb/s what could be the problem?

**Comment by norkunas** [comment]
Are you using `youtube-dl` binary or `yt-dlp`?

**Reply by array05** [reply]
`youtube-dl`

**Reply by norkunas** [reply, marked as answer]
Try the yt-dlp.
